charts/vh-video-web/values.dev.template.yaml

```diff
@@ -9,3 +9,5 @@ java:
     AZUREAD__REDIRECTURI: https://${SERVICE_FQDN}/home
     EJUDAD__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
     EJUDAD__REDIRECTURI: https://${SERVICE_FQDN}/home
+    DOM1__POSTLOGOUTREDIRECTURI: https://${SERVICE_FQDN}/logout
```

Comment:
@shaed-parkar should these changes be in here?

Reply:
ah that's a mistake when I was working on the wrong branch